scripts/build/companion_libs/220-ncurses.sh

```diff
@@ -49,7 +49,7 @@ do_ncurses_for_host() {
     # (i.e. in simple cross or native): ncurses may not be needed for
     # host, but we still need them on build to produce 'tic'.
     case "${CT_TOOLCHAIN_TYPE}" in
-        native|cross)   return 0;;
+        native)   return 0;;
```

Comment:
In case of the 'cross' type, build==host, so `do_ncurses_for_build` should have built ncurses already. Why wasn't it usable? It seems it is configured in the same way (the only differences are extra directories in cflags/ldflags - but those shouldn't matter as ncurses does not depend on any other library previously installed in buildtools/host_complibs).

In any case, the comment above this check needs to be updated.